network/src/lib.rs

```diff
@@ -427,7 +428,7 @@ impl PolkadotProtocol {
 	) {
 		match self.in_flight.remove(&(req_id, who.clone())) {
 			Some(mut req) => {
-				ctx.report_peer(who, cost::EXPECTED_MESSAGE);
+				ctx.report_peer(who, benefit::EXPECTED_MESSAGE);
```

Comment:
This is not necessarily a good response -- I think we should branch:
  - if `pov_block` is `None`, a small bump to reputation
  - if `pov_block` is `Some` and is good, a big bump to reputation
  - if `pov_block` is `Some` and is bad, a big hit to reputation.